prowler/lib/check/compliance.py

```diff
@@ -70,10 +71,10 @@ def update_checks_metadata_with_compliance(
                     "Recommendation": {"Text": "", "Url": ""},
                 },
                 "Categories": [],
-                "Tags": {},
                 "DependsOn": [],
                 "RelatedTo": [],
                 "Notes": "",
+                "Tags": {},
```

Comment:
Why is this here? The check metadata doesn't have a `Tags` key.

Reply:
True!